__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const app = require(`${__dirname}/../server/app.js`);
+const request = require('supertest');
+const db = require(`${__dirname}/../db/connection.js`);
+const seed = require(`${__dirname}/../db/seeds/seed.js`);
+const data = require(`${__dirname}/../db/data/test-data/index.js`)
```

Comment:
In the case of requiring in other files, the file path always starts on the file location, so you don't need the __dirname variable